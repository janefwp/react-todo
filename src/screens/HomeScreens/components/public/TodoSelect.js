import React from 'react'
import './Todocom.scss'
function TodoSelect(props) {
    return (
        <fieldset>
            <label>{props.label}</label>

                <props.as name={props.name} required={props.required} onChange={props.onChange}>
                    <option value="">Choose...</option>
                    <option value="css">css</option>
                    <option value="html">html</option>
                </props.as>

            
        </fieldset>
    )
}

export default TodoSelect
