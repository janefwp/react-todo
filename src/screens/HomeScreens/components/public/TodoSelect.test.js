import React from 'react';
import renderer from 'react-test-renderer';
import TodoSelect from './TodoSelect'

it('renders correctly when there is select type', () => {
    const tree = renderer.create(<TodoSelect as="select"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  