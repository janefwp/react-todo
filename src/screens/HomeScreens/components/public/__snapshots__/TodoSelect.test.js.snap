// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there is select type 1`] = `
<fieldset>
  <label />
  <select>
    <option
      value=""
    >
      Choose...
    </option>
    <option
      value="css"
    >
      css
    </option>
    <option
      value="html"
    >
      html
    </option>
  </select>
</fieldset>
`;
